lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'menu'
+require_relative 'keys'
+require 'twilio-ruby'
+
+class Takeaway
+  attr_reader :menu, :order, :order_total
+
+  def initialize(menu = Menu.new, order = {}, order_total = 0)
+    @menu = menu
+    @order = order
+    @order_total = order_total
```

Comment:
Does `order_total` need to be an instance variable? In this case, we're only using this variable in a single method, which suggests it can just be a local variable that exists only inside of the `total` method. In general, if a variable is used inside of only one method, it can be a local variable rather than an instance variable.

Having as little stored state as possible reduces the risk of another developer, for example, accidentally writing a method which will override `@order_total` with a nonsense value.